### PR TITLE
Add details for RHSA-2018-2654

### PR DESCRIPTION
**Pull request by ghost**
This PR updates the 3.6 release notes with details for RHSA-2018-2654. @ahardin-rh PTAL. Thanks.

To be released September 26, 2018

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@tmorriso-rh LGTM! Thanks! We will merge and cherry-pick when the release is available.

**Comment by tmorriso-rh** [comment]
@ahardin-rh I changed the information to RHSA. PTAL. Thanks.

File: release_notes/ocp_3_6_release_notes.adoc [hunk 1]
```diff
@@ -3757,3 +3757,28 @@ link:https://access.redhat.com/errata/RHBA-2018:2544[RHBA-2018:2544] advisory.
 To upgrade an existing {product-title} 3.5 or 3.6 cluster to this latest
 release, use the automated upgrade playbook. See
 xref:../install_config/upgrading/automated_upgrades.adoc#running-the-upgrade-playbook-directly[Performing Automated In-place Cluster Upgrades] for instructions.
+
+[[ocp-3-6-173-0-130]]
+=== RHBA-2018:2654 - {product-title} 3.6.173.0.130  Security, Bug Fix, and Enhancement Update
```
Comment (ahardin-rh) [inline release_notes/ocp_3_6_release_notes.adoc]: We need to update RHBA to RHSA in the heading

**Pull request merged**